tests/test_util_modules.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 import torch
 
+from hivemind import async_map, aiter, aenumerate, achain, anext, azip
```

Comment:
Better to make an explicit import from hivemind.utils

Reply:
good point, fixed